layouts/partials/landing_about.html

```diff
@@ -4,19 +4,14 @@ <h2>{{ .Params.about.title | markdownify }}</h2>
 		<p>{{ .Params.about.text | markdownify }}</p>
 	</div>
 	<div class="container">
-<!-- 		<div class="steps__inner">
-			{{ range $i, $box :=  .Params.about.box }}
-
-			{{ if eq (mod $i 4) 0 }}
-		</div> -->
 		<div class="steps__inner">
-			<!-- {{ end }} -->
-				<div class="step">
-					<div class="step__media"><img src="{{ $box.src }}" alt="{{ $box.alt }}" class="step__image" /></div>
-					<h4>{{ $box.title | markdownify }}</h4>
-					<p class="step__text">{{ $box.text | markdownify }}</p>
-				 </div>
-			<!-- {{ end }} -->
+			{{ range $i, $box :=  .Params.about.box }}
+			<div class="step">
+				<div class="step__media"><img src="{{ $box.src }}" alt="{{ $box.alt }}" class="step__image"></div>
+				<h4>{{ $box.title | markdownify }}</h4>
+				<p class="step__text">{{ $box.text | markdownify }}</p>
+			</div>
+			{{ end }}
```

Comment:
why did you uncomment that, and what it does?..

Reply:
It adds the individual boxes on the about page. I have no idea how it was working before, this part is required.

Reply:
**Be very careful here!**
This partial is referenced in this section: https://github.com/jellyfin/jellyfin-blog/blob/90f40d7aa00991411f102888de43714ab42a9a9e/layouts/index.html#L5

Or in other words, the “What is Jellyfin?” section on the home page. It’s my bad for leaving it in, but I recommend it be taken out instead.

When the page was initially created with the Hugo port, there: Movies, TV Shows, Music. In the meanwhile, the live page had Live TV & DVR added to it. When I brought that over to the port this code broke that layout, because it was expecting only three elements. I think there were also some padding concerns where things didn’t line up.

Anyway, make sure you test this change (which I trust you did).

Reply:
> I recommend it be taken out instead.

I think it will be useful with some minor wording changes.